includes/DomainCheckJob.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Miraheze\RequestSSL;
+
+use GenericParameterJob;
+use Job;
+use MediaWiki\MediaWikiServices;
+use User;
+
+class DomainCheckJob extends Job implements GenericParameterJob {
+	public function __construct( array $params ) {
+		parent::__construct( 'DomainCheckJob', $params );
+	}
+
+	public function run() {
+		$mwServices = MediaWikiServices::getInstance();
+		$requestSslManager = $mwServices->getService( 'RequestSSLManager' );
+		$requestSslManager->fromID( $this->params['requestID'] );
+		$isPointed = false;
+		$hookContainer = $mwServices->getHookContainer();
+		$hookContainer->run( 'RequestSSLDomainCheck', [&$requestSslManager, &$isPointed] );
```

Comment:
Regarding hooks, I have to register this hook on extension.json before usage, I don't think this will even work.

CreateWiki registers them in HookHandler like this. https://github.com/miraheze/CreateWiki/blob/3f8003973176c2225894dc39d73f1fe3a8e8275e/extension.json#L113

https://github.com/miraheze/CreateWiki/tree/master/includes/Hooks